File: CS/XtraPivotGrid_ADOMD/DataHelper.cs
using System;
using System.Reflection;
using DevExpress.XtraPivotGrid;

namespace XtraPivotGrid_ADOMD {
    public static class DataHelper {
        public static String GetCubePath() {
            Assembly assembly = Assembly.GetAssembly(typeof(PivotGridAdomdDataSource));
            Version version = assembly.GetName().Version;
            String textVersion = version.Major.ToString() + "." + version.Minor.ToString();
            String path = "C:\\Users\\Public\\Documents\\DevExpress 20" + textVersion +
                " Demos\\Components\\Data\\Northwind.cub";
            if (!System.IO.File.Exists(path)) {
                throw new Exception("Cannot find the Northwind cube. "  +
                    "Please ensure that the DXperience Suite is installed properly.");
            }
            return path;
        }
    }
}

File: VB/XtraPivotGrid_ADOMD/DataHelper.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Reflection
Imports DevExpress.XtraPivotGrid

Namespace XtraPivotGrid_ADOMD
	Public NotInheritable Class DataHelper
		Private Sub New()
		End Sub
		Public Shared Function GetCubePath() As String
			Dim [assembly] As System.Reflection.Assembly = System.Reflection.Assembly.GetAssembly(GetType(PivotGridAdomdDataSource))
			Dim version As Version = [assembly].GetName().Version
			Dim textVersion As String = version.Major.ToString() & "." & version.Minor.ToString()
			Dim path As String = "C:\Users\Public\Documents\DevExpress 20" & textVersion & " Demos\Components\Data\Northwind.cub"
			If (Not System.IO.File.Exists(path)) Then
				Throw New Exception("Cannot find the Northwind cube. " & "Please ensure that the DXperience Suite is installed properly.")
			End If
			Return path
		End Function
	End Class
End Namespace